cmd/kor/root.go

```diff
@@ -12,7 +13,9 @@ var rootCmd = &cobra.Command{
 	Long: `kor is a CLI to to discover unused Kubernetes resources 
 	kor can currently discover unused configmaps and secrets`,
 	Run: func(cmd *cobra.Command, args []string) {
-
+		if len(args) == 0 {
```

Comment:
nit: it would be better to set the `Args` property on the `cobra.Command`, something like 
```
Args: cobra.MinimumNArgs(1),
```
would yield  the same result.

Reply:
@luisdavim Sounds good. Can you open an MR for this?